src/view/iView.cxx

```diff
@@ -1,4 +1,3 @@
 #include "iView.hxx"
 
 iView::~iView() = default;
```

Comment:
Consider moving it to the hxx file.

Reply:
[It's better to make a pure virtual destructor not inline](https://stackoverflow.com/questions/4998344/why-must-a-pure-virtual-destructors-implementation-be-empty-and-should-it-be-i)